portfolio/src/main/webapp/index.js

```diff
@@ -12,17 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+"use strict";
+import {setToNone} from "./utils.js";
+import {Tab} from "./tab.js";
+import {SlideShow} from "./slideshow.js";
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+window.onload = function() {
+  // Allow all imported functions to be in the scope of "window".
+  window.tab = new Tab();
+  window.slideShow = new SlideShow();
+  
+  // Set up default tab and slideshow.
+  tab.displayDefaultSection();
```

Comment:
The reason referencing tab this way works is because the interpreter will check the current scope for something called "tab" and work its way up the scope chain until it finds the first variable that matches. This is error prone and can lead to unexpected behavior.

Option 1 (recommended): make a const tab = new Tab() in this class.
Option 2: reference it as window.tab every time.

Whatever option you go with applies to slideshow as well.